uber/templates/registration/form.html

```diff
@@ -382,4 +385,28 @@
 
 </div>
 
+<script>
+var confirmAction = function(){
+    var check = confirm("Are you sure you'd like to check out this attendee?");
+    if (check){
+        $.ajax({
+            "url":"check_out_attendee_in_error",
+            "type":"POST",
+            "dataType":"json",
+            "data":{
+                "id":"{{ attendee.id }}",
+                "csrf_token":"{{ c.CSRF_TOKEN }}"
+            },
+            success: function(message){
+                toastr.info(message);
+                setTimeout(function(){
+                    location.reload(true);
+                }, 1500);
```

Comment:
I don't think I like the approach of displaying and message and then reloading the page, which would confuse me as a user if I didn't already know it was programmed to do this.  Presumably this was done so that we could display the correct information.  However, if that's the goal, a better approach would be to handle this like we do on the check-in page and dynamically update the relevant pieces on the page.  I believe there are only two things which would need to be updated:
- The text saying they're checked in.
- Possibly their badge number depending on whether they had a preassigned badge type.

We can handle this by putting divs with ids around that data.  This method could then be changed to expect back a JSON object that looks like this
```
{
    "new_badge": "Attendee (Volunteer)",
    "message": "Attendee un-checked-in."
}
```

Then here in this success handler, we could do something like

```
$('#checked_in').text('no');
$('#badge').text(response.new_badge);
```